### ISSUE TITLE
Cannot pick a time when creating new event in month view

### ISSUE THREAD

**Issue by paulchen** [nextcloud/calendar#2983]
When creating a new calendar event, I'm unable to "Pick a time" in the overlay because the time picker is not shown:

![image](https://user-images.githubusercontent.com/404840/112755932-2ec5df80-8fe3-11eb-9544-b2826eb8a521.png)

However, the time picker is working
- when modifying an already existing event or
- after clicking on "More" to open the sidebar.

I observed this behaviour on two different instances of Nextcloud.

This stopped working a few days ago, perhaps with the update of the calendar app to 2.2.0.

### Steps to reproduce
1. Click into the calendar for creating a new event.
2. De-select "All day"
3. Click into one of the input fields for start or end time.
4. Click pick a time.

### Expected behaviour
The time picker is shown and I'm able to pick a time.

### Actual behaviour
The time picker is not shown and thus I'm unable to pick a time.

### Calendar app
**Calendar app version:** 2.2.0

**CalDAV-clients used:** not revelant

### Client configuration
**Browser:** Firefox 87.0 (64-bit), Chromium "Version 89.0.4389.90 (Offizieller Build) Arch Linux (64-Bit)"

**Operating system:** Arch Linux

### Server configuration

**Operating system**: Debian 10.9

**Web server:** Apache 2.4.38 with libapache2-mod-php7.3, nginx 1.14.2 with php7.3-fpm

**Database:** MariaDB 10.3.27

**PHP version:** PHP 7.3.27-1~deb10u1

**Nextcloud Version:** 20.0.8

**Updated from an older installed version or fresh install:** older installed version

**List of activated apps:**

Instance 1:

```
Enabled:
  - accessibility: 1.6.0
  - activity: 2.13.4
  - admin_audit: 1.10.0
  - calendar: 2.2.0
  - cloud_federation_api: 1.3.0
  - contacts: 3.5.1
  - contactsinteraction: 1.1.0
  - dashboard: 7.0.0
  - dav: 1.16.2
  - federatedfilesharing: 1.10.2
  - files: 1.15.0
  - files_pdfviewer: 2.0.1
  - files_rightclick: 0.17.0
  - files_sharing: 1.12.2
  - files_trashbin: 1.10.1
  - files_versions: 1.13.0
  - files_videoplayer: 1.9.0
  - logreader: 2.5.0
  - lookup_server_connector: 1.8.0
  - nextcloud_announcements: 1.9.0
  - notifications: 2.8.0
  - oauth2: 1.8.0
  - password_policy: 1.10.1
  - photos: 1.2.3
  - previewgenerator: 3.1.1
  - privacy: 1.4.0
  - provisioning_api: 1.10.0
  - recommendations: 0.8.0
  - serverinfo: 1.10.0
  - settings: 1.2.0
  - support: 1.3.0
  - survey_client: 1.8.0
  - text: 3.1.0
  - theming: 1.11.0
  - twofactor_backupcodes: 1.9.0
  - updatenotification: 1.10.0
  - user_status: 1.0.1
  - viewer: 1.4.0
  - weather_status: 1.0.0
  - workflowengine: 2.2.0
Disabled:
  - comments
  - documentserver_community
  - encryption
  - federation
  - files_external
  - firstrunwizard
  - onlyoffice
  - sharebymail
  - systemtags
  - user_ldap
```

Instance 2:

```
Enabled:
  - accessibility: 1.6.0
  - activity: 2.13.4
  - admin_audit: 1.10.0
  - calendar: 2.2.0
  - cloud_federation_api: 1.3.0
  - contacts: 3.5.1
  - contactsinteraction: 1.1.0
  - dashboard: 7.0.0
  - dav: 1.16.2
  - federatedfilesharing: 1.10.2
  - files: 1.15.0
  - files_markdown: 2.3.3
  - files_pdfviewer: 2.0.1
  - files_sharing: 1.12.2
  - files_trashbin: 1.10.1
  - files_versions: 1.13.0
  - files_videoplayer: 1.9.0
  - firstrunwizard: 2.9.0
  - logreader: 2.5.0
  - lookup_server_connector: 1.8.0
  - nextcloud_announcements: 1.9.0
  - notifications: 2.8.0
  - oauth2: 1.8.0
  - password_policy: 1.10.1
  - photos: 1.2.3
  - privacy: 1.4.0
  - provisioning_api: 1.10.0
  - recommendations: 0.8.0
  - serverinfo: 1.10.0
  - settings: 1.2.0
  - sharebymail: 1.10.0
  - survey_client: 1.8.0
  - systemtags: 1.10.0
  - text: 3.1.0
  - theming: 1.11.0
  - twofactor_backupcodes: 1.9.0
  - updatenotification: 1.10.0
  - user_status: 1.0.1
  - viewer: 1.4.0
  - weather_status: 1.0.0
  - workflowengine: 2.2.0
Disabled:
  - checksum
  - comments
  - deck
  - documentserver_community
  - encryption
  - federation
  - files_external
  - files_rightclick
  - notes
  - onlyoffice
  - polls
  - quicknotes
  - support
  - user_ldap
```

**Nextcloud configuration:**

Instance 1:

```
{
    "system": {
        "debug": false,
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "20.0.8.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "versions": "false",
        "maintenance": false,
        "forcessl": true,
        "theme": "",
        "loglevel": 0,
        "trusted_domains": [
            "cloud.rueckgr.at",
            "cloud.rueckgr.at"
        ],
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Memcached",
        "memcached_servers": [
            [
                "localhost",
                11211
            ]
        ],
        "cache_path": "",
        "trashbin_retention_obligation": "auto",
        "overwrite.cli.url": "https:\/\/cloud.rueckgr.at",
        "mail_smtpauthtype": "LOGIN",
        "updater.release.channel": "stable",
        "mysql.utf8mb4": true,
        "app_install_overwrite": [
            "calendar",
            "keeweb"
        ]
    }
}
```

Instance 2:

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.huntu.at"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "20.0.8.1",
        "overwrite.cli.url": "https:\/\/cloud.huntu.at",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpsecure": "tls",
        "mail_smtpport": "587",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "theme": "",
        "loglevel": 2,
        "mysql.utf8mb4": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 1.5
        },
        "updater.release.channel": "stable",
        "app_install_overwrite": [
            "calendar"
        ]
    }
}
```

#### Web server error log (e.g. /var/log/apache)

Nothing relevant found in access.log or error.log.

#### Log file (data/nextcloud.log)

Nothing relevant found.

#### Browser log

Browser console:
```
index.js:46 No OC found
t.getLanguage @ index.js:46
value @ gettext.js:45
(anonymous) @ l10n.js:3
(anonymous) @ main.js?v=51d34b65-12:1
n @ bootstrap:19
(anonymous) @ main.js?v=51d34b65-12:1
n @ bootstrap:19
(anonymous) @ main.js:1
(anonymous) @ main.js?v=51d34b65-12:1
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ main.js?v=51d34b65-12:1
jquery-migrate.min.js:2 JQMIGRATE: Migrate is installed, version 1.4.1
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
o @ jquery.js:9789
0 @ jquery.js:26
n @ bootstrap:19
671 @ files_client.js?v=51d34b65-12:1
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ files_client.js?v=51d34b65-12:1
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
o @ jquery.js:9792
0 @ jquery.js:26
n @ bootstrap:19
671 @ files_client.js?v=51d34b65-12:1
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ files_client.js?v=51d34b65-12:1
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
mounted @ App.vue:117
Ue @ vue.runtime.esm.js:1854
Kt @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
w @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
e._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
pn.get @ vue.runtime.esm.js:4479
pn @ vue.runtime.esm.js:4468
e @ vue.runtime.esm.js:4073
Ln.$mount @ vue.runtime.esm.js:8415
e._init @ vue.runtime.esm.js:5018
Ln @ vue.runtime.esm.js:5085
(anonymous) @ main.js:37
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ notifications-main.js?v=51d34b65-12:1
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
mounted @ App.vue:121
Ue @ vue.runtime.esm.js:1854
Kt @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
w @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
e._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
pn.get @ vue.runtime.esm.js:4479
pn @ vue.runtime.esm.js:4468
e @ vue.runtime.esm.js:4073
Ln.$mount @ vue.runtime.esm.js:8415
e._init @ vue.runtime.esm.js:5018
Ln @ vue.runtime.esm.js:5085
(anonymous) @ main.js:37
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ notifications-main.js?v=51d34b65-12:1
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
mounted @ App.vue:121
Ue @ vue.runtime.esm.js:1854
Kt @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
w @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
e._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
pn.get @ vue.runtime.esm.js:4479
pn @ vue.runtime.esm.js:4468
e @ vue.runtime.esm.js:4073
Ln.$mount @ vue.runtime.esm.js:8415
e._init @ vue.runtime.esm.js:5018
Ln @ vue.runtime.esm.js:5085
(anonymous) @ main.js:37
n @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ notifications-main.js?v=51d34b65-12:1
App.vue:296 Notifications permissions granted
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
(anonymous) @ script.js?v=51d34b65-12:492
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
ie @ globals.js:60
get @ globals.js:91
(anonymous) @ files.js?v=51d34b65-12:122
ConsoleLogger.js:46 [INFO] calendar: 
Initial settings:
	- AppVersion: 2.2.0
	- EventLimit: true
	- FirstRun: false
	- ShowWeekNumbers: true
	- ShowTasks: true
	- ShowWeekends: true
	- SkipPopover: false
	- SlotDuration: 00:30:00
	- TalkEnabled: false
	- TasksEnabled: false
	- Timezone: Europe/Vienna
 {app: "calendar", uid: "paulchen"}
ConsoleLogger.js:42 [DEBUG] calendar: The calendar app is using version 2.2019c of the timezone database {app: "calendar", uid: "paulchen"}
session-heartbeat.js:97 session heartbeat polling started
public.js:30 Files_PDFViewer initialized for public page {isPublicPage: null, canDownload: true, isSecureViewerAvailable: false}
/index.php/apps/calendar/dayGridMonth/now:1 [DOM] Password forms should have (optionally hidden) username fields for accessibility: (More info: https://goo.gl/9p2vKq) <form id=​"sudo-login-form" class=​"hidden" method=​"POST">​<label>​…​</label>​<input class=​"confirm" value=​"Bestätigen" type=​"submit">​</form>​
[Violation] Forced reflow while executing JavaScript took 96ms
ConsoleLogger.js:42 [DEBUG] unified-search: Unified Search initialized with the following providers {0: {…}, 1: {…}, 2: {…}, 3: {…}, 4: {…}, 5: {…}, app: "unified-search", uid: "paulchen"}0: {__ob__: we}1: {__ob__: we}2: {__ob__: we}3: {__ob__: we}4: {__ob__: we}5: {__ob__: we}app: "unified-search"uid: "paulchen"__proto__: Object
ConsoleLogger.js:46 [INFO] calendar: Updated moment locale: de {app: "calendar", uid: "paulchen"}
ConsoleLogger.js:42 [DEBUG] calendar: Current user principal is /remote.php/dav/principals/users/paulchen/ {app: "calendar", uid: "paulchen"}
[Violation] Forced reflow while executing JavaScript took 77ms
[Violation] Forced reflow while executing JavaScript took 44ms
[Violation] 'setInterval' handler took <N>ms
[Violation] 'setInterval' handler took <N>ms
[Violation] 'setInterval' handler took <N>ms
[Violation] 'setInterval' handler took <N>ms
[Violation] 'setInterval' handler took <N>ms
[Violation] 'setInterval' handler took <N>ms```

**Comment by m8ichael** [comment]
This seems to be a general problem, and I wonder why such obvious problems are not detected in the testing phase before a new release is released.

**Comment by miaulalala** [comment]
Confirmed.

**Comment by mattin4d** [comment]
On Nextcloud 21 I at least see the am/pm selection but still no time selection
![image](https://user-images.githubusercontent.com/1329241/113885842-c9fa4a00-9785-11eb-9c25-d9870e05f7fa.png)

**Comment by raimund-schluessler** [comment]
This only happens in month view, which might be a reason why it wasn't noticed quick enough. As a workaround for now, you can just switch to week or day view and create your event there.

**Comment by yan12125** [comment]
> This stopped working a few days ago, perhaps with the update of the calendar app to 2.2.0.

Reverting to 2.1.3 makes the time picker working again for me. I may try bisecting.

**Comment by yan12125** [comment]
By bisecting this repo, it seems the issue is from nextcloud-vue...
```
7d372b32eb6a646c0b649a80f7766bbd882df88e is the first bad commit
```
7d372b32eb6a646c0b649a80f7766bbd882df88e updates nextcloud-vue from 2.6.5 to 2.9.0.

**Comment by tcitworld** [comment]
2.2.2 has been released with the fix.

**Comment by paulchen** [comment]
After updating the Calendar app to the newly released version 2.2.2, I can confirm that the time selection now works as expected on all my instances of Nextcloud:

![image](https://user-images.githubusercontent.com/404840/119666133-5dbacf00-be35-11eb-833c-250be0e42e4d.png)

Thanks for fixing this problem!
